src/server/serverSetup.js

```diff
@@ -158,6 +180,18 @@ fluid.defaults("sjrk.storyTelling.server.app.storyTellingHandlers", {
             "route": "/admin/deleteStory/:id",
             "method": "get"
         },
+        testDataHandler: {
```

Comment:
What is `testDataHandler` for? There isn't a /testData sub-directory in the sjrk root directory.

Reply:
There's a testData directory in the /tests/ directory, which contains some files used in both the node and browser tests. The contents are served so the browser tests can refer to them. I didn't want to serve/expose the entire /tests/ directory, but considering we're keeping the node and browser tests separate, I didn't want to keep the testData directory in one or the other of those two locations.

Reply:
Got it. Thanks.